packages/lib/versionInfo.test.ts

```diff
@@ -0,0 +1,30 @@
+import { _ } from './locale';
+import { getPluginsArr } from './versionInfo';
+import PluginService from '@joplin/lib/services/plugins/PluginService';
+
+const plugins = getPluginsArr(PluginService.instance().plugins);
+
+describe('versionInfo', function() {
+
+	it('should return an array', (()=>{
+		expect(Array.isArray(plugins)).toBe(true);
+	}));
+
+	it('should return an empty array', (() => {
+		expect(plugins.length).toBe(0);
+	}));
+
+	it('should not push plugin to body if no plugin is found',(()=>{
```

Comment:
This test is not really doing anything meaningful as it never calls any code that is used by Joplin.

Reply:
Can you suggest any test case that I should Add?
I am having confusion about testing the file without any plugins in the test environment.

Reply:
Essentially you have a function from Plugins to string - this is all you need to test. When running in Joplin Plugins will be provided by the PluginService but in your test you can set the value manually and use it for testing.

Reply:
Added New Tests